renpy/curry.py

```diff
@@ -73,6 +70,42 @@ def __hash__(self):
         return self.hash
 
 
+class Partial(functools.partial):
+    """
+    Stores a callable and some arguments. When called, calls the
+    callable with the stored arguments and the additional arguments
+    supplied to the call.
+    """
+
+    __slots__ = ("hash",)
```

Comment:
We probably don't want slots here, as that means that something that  worked before: 

```
t1 = renpy.curry(test):
t1.predict = predict_t1 

Would stop working. 
```

Reply:
I would still work because functools.partial includes "\_\_dict__" in the \_\_slots__